### ISSUE TITLE
Can't go back out of YouTube (with remote back)

### ISSUE THREAD

**Issue by dnarcese** [mozilla-mobile/firefox-tv#1542]
## Steps to reproduce
1. Click Google search tile
2. Open overlay and click YouTube tile
3. Click back button on remote several times to try and get back to Google
### Expected behavior
Can get back to Google
### Actual behavior
Cannot go back to Google from the remote back button. Focus stays on the 'Home' icon of YouTube.
### Device information

* Fire TV device: 4k
* Firefox version: 3.1

**Comment by dnarcese** [comment]
Blocked on #1543

**Comment by mcomella** [comment]
> Blocked on #1543

I disagree – see that issue. I have a functioning WIP that fixes this issue, possibly fragile, in https://github.com/mcomella/firefox-tv/tree/i773-youtube-back

My WIP only goes back in the browser stack: it won't pop into the fragment back stack or close the app.

**Comment by dnarcese** [comment]
#1576 was merged into releases/3.1.3.
